documentation/docusaurus.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  plugins: [
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'docs-template',  // Usually your repository's name, in this case docs-template
+        path: path.resolve(__dirname, 'docs'),
+        routeBasePath: 'docs-template', // Usually your repository's name, in this case docs-template
+        sidebarPath: path.resolve(__dirname, 'sidebars.js'),
+        editUrl: 'https://YOURREPOSITORYURL/edit/YOURDESIREDBRANCHNAME/documentation',// Example: https://github.com/iotacommunity/docs-template/edit/production/documentation
+        remarkPlugins: [require('remark-code-import'), require('remark-import-partial')],// You can add any remark or rehype extensions you may need here 
```

Comment:
Related to the earlier comment about preferring Docusaurus Markdown and MDX capabilities over using remakr and rehype plugins. The code import plugin might be one of the exceptions, just like the math plugins. They provide functionality Docusaurus does not provide. We should consider to add it to our default config though, as all repos with code examples rely on it.

Reply:
```suggestion
```

Reply:
the import partial and code import are common to most repos.  It'd be great to add them to the default config, but until we do it's probably best to leave them in the template.  Though we can probably add them to the default before anyone uses this.  Let's discuss it tomorrow :)

Reply:
Keep in mind that it is fine to iterate. We can always open an issue and look at it later. 
We really need to focus on the important stuff now. But yeah, let's discuss tomorrow